subsys/bluetooth/controller/ll_sw/ull_llcp_remote.c

```diff
@@ -197,6 +197,8 @@ static void rr_act_run(struct ll_conn *conn)
 	struct proc_ctx *ctx;
 
 	ctx = rr_peek(conn);
+	LL_ASSERT(ctx != NULL);
```

Comment:
rr_act_run is already guarded at callsite to have a valid context at the head of the queue